### PR TITLE
Add Python QuickStart

### PR DESCRIPTION
**Pull request by JonasVautherin**

### CODE REVIEW

File: en/python/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+```
+commander takeoff
+commander land
+```
+
+The simulated drone should takeoff and land. If it doesn't, it may mean that SITL is not ready, or that there is a problem.
+
+## Take off from MAVSDK
+
+When we know that the simulator is ready, we can open an `apython` REPL:
```
Comment (hamishwillee) [inline en/python/quickstart.md:37]: What does this mean? Open a file? i.e. what am I seeing here?

Comment (hamishwillee) [reply en/python/quickstart.md:37]: Ignore, answered before.

**Comment by hamishwillee** [comment]
Love it. I have subedited and merged. Check it out - fixes can be a post process.

**Review by julianoes — COMMENTED** [review]
Nice, thanks @JonasVautherin!

**Pull request merged**